docs/source/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,19 @@
+{% extends "!autosummary/class.rst" %}
+
+{#
+An autosummary template to exclude the class constructor (__init__)
+which doesn't contain any docstring in Optuna.
+#}
+
+{% block methods %}
+   {% set methods = methods | select("ne", "__init__") | list %}
+   {% if methods %}
+   .. rubric:: Methods
```

Comment:
Can I ask a question? 
I'm a bit curious if we don't need to use `{{ _('Methods') }}` as can be seen in the original template as follows:

```rst
.. autoclass:: {{ objname }}

   {% block methods %}
   .. automethod:: __init__

   {% if methods %}
   .. rubric:: {{ _('Methods') }}  {# Here #}

   .. autosummary::
   {% for item in methods %}
      ~{{ name }}.{{ item }}
   {%- endfor %}
   {% endif %}
   {% endblock %}
```

(c.f., https://github.com/sphinx-doc/sphinx/blob/master/sphinx/ext/autosummary/templates/autosummary/class.rst#L11)

Reply:
Ah, I found that this notation was related to translation. `_()` is equivalent to `gettext()`, which translates a single string. We may integrate the [Chinese docs](https://zh-cn.optuna.org/) into the English docs somehow, so could you use `gettext` here?
https://jinja.palletsprojects.com/en/2.11.x/templates/#i18n

Reply:
Currently, we do not use the `i18n` feature, so I think this modification is optional.
Please ignore my comment, if you have any troubles to introduce `gettext()`.

Reply:
I just replaced `Methods` to `{{ gettext('Methods') }}`, and encountered the following error.
```
    getattr(obj, "unsafe_callable", False) or getattr(obj, "alters_data", False)
jinja2.exceptions.UndefinedError: 'gettext' is undefined
```

Sphinx uses the [Jinja templating engine](https://www.sphinx-doc.org/en/master/templating.html) and we have to introduce an extension via Sphinx in some way. And unfortunately, I'm afraid that I can't find a way to do this.

How about making this modification an another issue, and thinking about it when we work on translations?

Reply:
@upura Thank you for your investigation. That makes sense. I'll work on it when we introduce multi-lingual documentation.